VideoLocker/res/values/strings.xml

```diff
@@ -28,8 +28,15 @@
     <string name="share_application_picture_url">https://www.edx.org/sites/default/files/theme/edx-logo-header.png</string>
     <string name="share_application_description">Free online courses</string>
 
-    <!-- Settings Page -->
+    <!-- Sharing -->
+
+    <!-- Share menu title -->
+    <string name="share_via">Share via&#8230;</string>
+    <!-- Share course -->
+    <string name="share_course">I’m taking a course on {platform_name}! Check it out:</string>
+    <string name="share_icon">&#xf1e0;</string>
```

Comment:
It's a custom character, represented by font-awesome's icon font.

Anyway I don't think this is used anymore.